docs/static/ls-ls-native.asciidoc

```diff
@@ -36,15 +36,13 @@ input {
 
 In order to obtain the best performance when sending data from one Logstash to another, the data is batched and compressed. As such, the upstream Logstash (the sending Logstash) only needs to be concerned about configuring the receiving endpoint with these options:
 
-* `hosts` - The receiving Logstash and port. If no port specified, 9800 will be used.
-
-NOTE: In the future, {ls} will support multiple output hosts.
```

Comment:
I think the note should highlight how we do HA, and what multiple hosts does. 

e.g. NOTE: When setting multiple hosts, Logstash will load balance the communication across the receiving endpoint. The loadbalancing strategy is [...]